packages/web-components/src/component-mixins/cta/cta.ts

```diff
@@ -322,6 +322,57 @@ const CTAMixin = <T extends Constructor<HTMLElement>>(Base: T) => {
       if (ctaType === CTA_TYPE.VIDEO && this.offsetWidth > 0) {
         this._updateVideoThumbnailUrl();
       }
+
+      // Once the eventRequestVideoData finishes and has set the videoName,
+      // check for the URL trigger meant to fire eventRunAction.
+      if (
+        ctaType === CTA_TYPE.VIDEO &&
+        href &&
+        changedProperties.has('videoName') &&
+        videoName
+      ) {
+        this._checkUrlVideoTrigger();
+      }
```

Comment:
This works and is _probably_ safe, but this setup also allows the possibility of rouge lightboxes if the video name were to be changed for any reason.

1. Open [demo](https://sandbox.andy-blum.com/ibm/auto-open-lightbox/multiple-ctas.html#cta-video-0_ibuqxqbe)
2. Close the lightbox after it opens
3. Inspect the first link-with-icon
4. Manually alter the video-name property

I wonder if this logic would be better in `firstUpdated`?

```ts
class Element {
  firstUpdated(changedProperties) {
    if (document.readyState !== 'complete') {
      document.addEventListener('load', this._checkUrlVideoTrigger.bind(this));
    } else {
      this._checkUrlVideoTrigger();
    }
  }
}
```